scipy/sparse/linalg/isolve/iterative/test.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import division, print_function, absolute_import
 
 from scipy import *
+from numpy import *
```

Comment:
Maybe clean these two imports, too, while you're at it?

Reply:
Done